src/hooks/usePersistentAspectRatio.ts

```diff
@@ -23,26 +23,31 @@ type Result = {
 };
 
 export default function usePersistentAspectRatio(url: string): Result {
+  if (url === null) url = 'default';
```

Comment:
not a fan of magic words and overriding arguments. This is dirty and I don't want to establish this as an acceptable pattern